compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIELinalgFunctionOutlining.cpp

```diff
@@ -88,6 +100,12 @@ void AMDAIELinalgFunctionOutliningPass::runOnOperation() {
   DenseMap<Operation *, std::string> computeOpToOutlinedFuncMap;
   SmallVector<Operation *> toBeErased;
   moduleOp.walk([&](linalg::LinalgOp computeOp) {
+    if (mustNotOutline(computeOp))
+      return WalkResult::skip();
+    else if (!mustOutline(computeOp)) {
```

Comment:
```suggestion
    if (mustNotOutline(computeOp)) {
      return WalkResult::skip();
    } else if (!mustOutline(computeOp)) {
```
See style guide: https://google.github.io/styleguide/cppguide.html#Formatting_Looping_Branching